app/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
 <meta charset="utf-8">
-<meta http-equiv="Content-Security-Policy" content="default-src 'self' ; style-src 'self' 'unsafe-inline' ; connect-src * ; object-src 'none' ; referrer no-referrer ;">
+<meta http-equiv="Content-Security-Policy" content="default-src 'self' ; style-src 'self' 'unsafe-inline' ; connect-src * ; object-src 'none' ; img-src * ; referrer no-referrer ;">
```

Comment:
__question:__ I need to investigate this further, what I need is image-src that points to foxbox (foxbox provides endpoint that renders jpeg directly). What restrictions\rules do we have here?